src/current/_includes/releases/v23.2/v23.2.0-alpha.1.md

```diff
@@ -421,6 +471,13 @@ Release Date: September 23, 2023
     [#101352][#101352]
 
 - CockroachDB now prevents setting `max_range_size` below the `COCKROACH_MIN_RANGE_MAX_BYTES` environment variable, which defaults to 64 MiB (half of the default minimum range size). [#96725][#96725]
+- Fixed a bug that could occasionally cause schema change jobs, such as table or index drops, to appear stuck in state "waiting for MVCC GC" for much longer than expected. The fix only applies to future schema changes; to process existing stuck jobs, manually force-enqueue the relevant ranges in the MVCC GC queue from the DB Console's [Advanced Debug](https://www.cockroachlabs.com/docs/{{ page.version.version }}/ui-debug-pages) page. [#110078][#110078]
```

Comment:
The second sentence here, I would maybe break up:

> The fix only applies to future schema changes. To process existing stuck jobs, manually force-enqueue the relevant ranges in the MVCC GC queue from the DB Console's [Advanced Debug](https://www.cockroachlabs.com/docs/{{ page.version.version }}/ui-debug-pages) page.

Reply:
Additionally, `s/{{ page.version.version }}/v23.2/g`.